### ISSUE TITLE
ibm-property-casing-convention doesn't work

### ISSUE THREAD

**Issue by fenuks** [IBM/openapi-validator#752]
For this minimal example taken from [rule docs](https://github.com/IBM/openapi-validator/blob/main/docs/ibm-cloud-rules.md#ibm-property-casing-convention):

```yml
openapi: 3.0.0

components:
  schemas:
    Thing:
      type: object
      properties:
        thingId:
          type: string
        thingType:
          type: string
```

`lint-openapi spec.yml` doesn't report that casing is wrong.

I'm using `openapi-validator` v1.35.2 (latest at the moment).
